### PR TITLE
provide internal paths, fixes #24

### LINKED ISSUE #24: references to data-root broken
while looking at #21 I came across those lines https://github.com/Edirom/MerMEId/blob/9e5e1d1bab0da69b301415cd0c0a689b40016ab6/data/controller.xql#L21

and https://github.com/Edirom/MerMEId/blob/9e5e1d1bab0da69b301415cd0c0a689b40016ab6/data/controller.xql#L43   

Both are addressing resources within the database so should be written to use internal paths (namely `$data-root`) rather than external paths which may vary depending on the deployment paths.

### PR DESCRIPTION
**Pull request by peterstadler**
replace internal database paths with proper variables

### CODE REVIEW

**Comment by simar0at** [comment]
Looks good. But just read the code, did not test.

**Review by MauPalantir — APPROVED** [review]

**Comment by peterstadler** [comment]
Hi @MauPalantir, if you tested this with your setup I'd be glad if you just hit the "merge" button :)

**Review by MauPalantir — COMMENTED** [review]
@peterstadler hmm, is the removal of line 44 intentional? should I test that too?

**Review by MauPalantir — CHANGES_REQUESTED** [review]
because I didn't notice it until now, and when I port that one too to my dev env, it gives me an error in eXide.

**Comment by simar0at** [comment]
removal of line 44 is most probably nonsens.

File: data/controller.xql [hunk 1]
```diff
@@ -40,8 +40,7 @@ if (ends-with($exist:resource, ".xml")) then
                 <param name="exist:stop-on-warn" value="no"/>
                 <param name="exist:stop-on-error" value="no"/>
 </parameters>, <attributes></attributes>, "method=xml media-type=application/xml"),
-        $saved := xmldb:store("/db"||$exist:prefix||$exist:controller||string-join(tokenize($exist:path, '/')[position() != last()], '/'), $exist:resource, $filtered)
-    return doc($saved)
```
Comment (simar0at) [inline data/controller.xql:44]: This is most probably an error.

**Review by simar0at — CHANGES_REQUESTED** [review]
Please test again with the return statement.

File: data/controller.xql (hunk 1 above)
Comment (peterstadler) [reply data/controller.xql:44]: Yes, thanks for spotting. Re-added it in  815a14c

**Review by simar0at — APPROVED** [review]
This looks good. I don't have time to test it. Perhaps Zsofia can.

**Review by MauPalantir — APPROVED** [review]

**Pull request merged**